src/_scss/pages/dashboard/table/_dashboardPagination.scss

```diff
@@ -0,0 +1,50 @@
+.dashboard-table-pagination {
+    @import "../../../sharedComponents/paginator";
+    @import "../../../sharedComponents/limitSelector";
+    @include display(flex);
+
+    .table-paginator {
+        li {
+            line-height: 1em;
+            font-size: rem(12);
+
+            button {
+                border-color: $color-gray-lighter;
+                color: #0074be;
+                height: rem(26);
+                width: rem(26);
+                padding: rem(2);
+            }
+
+            svg {
+                &.fa-ellipsis-h {
+                    font-size: rem(13);
+                }
+            }
+    
+            &.active {
+                button {
+                    background-color: #00a6d2;
+                    border-color: #00a6d2;
+                    color: $color-white;
+                    font-weight: initial;
+                }
+            }
+
+            &.page-arrow {
+                button {
+                    padding: rem(2);
+
+                    .usa-da-icon {
+                        height: rem(20);
+                        width: rem(20);
+
+                        svg{
+                            fill: #0074be;
```

Comment:
This can become `$color-primary` (just double checked with @davidsbill that when colors in the mockup are very similar to our existing variables, we should go with the colors defined in [colors.scss](https://github.com/fedspendingtransparency/data-act-broker-web-app/blob/dev/src/_scss/abstract/variables/colors.scss))